### DISCUSSION TITLE
How can I integrate my test cases written in WebdriverIO-Cucumber-Typescript framework with HP ALM?

### DISCUSSION THREAD

**Discussion by vikrantpunde** [webdriverio/webdriverio#11761] [Q&A]
We are using WebdriverIO-Cucumber-Typescript to write our test cases and executing it in local machine & also through Jenkins CICD. We want to integrate our automation test cases with HP ALM tool for test management and execution. Is there any way to integrate webdriverIO-Cucumber test automation test cases with HP ALM?

**Comment by christian-bromann** [comment]
@vikrantpunde I personally never worked with HP ALM and hear it for the first time. I don't know of any integration to it but wouldn't be surprised if there is one. I would reach out to HP as they should provide you guidance.
